CODEOWNERS

```diff
@@ -13,10 +15,6 @@ CODEOWNERS @tt-rkim
 INSTALLING.md @tt-rkim @TT-billteng
 METALIUM_GUIDE.md @davorchap
 
-# Dependencies
-
-third_party/ @tt-rkim @TT-billteng
```

Comment:
This rule had no impact, and is overruled later by:

`tt_metal/ @abhullar-tt @pgkeller @aliuTT @tt-aho @tt-dma @tt-asaigal @ubcheema`